### ISSUE TITLE
meow-line does not work as expected in emacs 28.1

### ISSUE THREAD

**Issue by artistu** [meow-edit/meow#235]
When I press "x" (binded to meow-line) I expect to "select the current line" as described in **meow-tutor**. Instead the cursor does not move and the message "The mark is not active now" is displayed in the minibuffer. I'm aware about the note "I encountered a weird command behavior in normal state" in the FAQ but I'm not sure how it would apply to my case as there is not a **meow--kbd-*** variable (as far as I'm aware) that affects that command. 

Is this a consequence of defining the **(meow-setup)** function? Is there a default that can be set automatically?

Thanks for any assistance you can provide.

My meow-setup is the following:
```
(use-package meow
        :straight (meow :type git
                                                                        :host github
                                                                        :repo "meow-edit/meow")
        :demand t
        :config
        (defun meow-setup ()
                (setq meow-cheatsheet-layout meow-cheatsheet-layout-qwerty)
                (meow-motion-overwrite-define-key
                 '("j" . meow-next)
                 '("k" . meow-prev)
                 '("<escape>" . ignore))
                (meow-leader-define-key
                 ;; SPC j/k will run the original command in MOTION state.
                 '("j" . "H-j")
                 '("k" . "H-k")
                 ;; Use SPC (0-9) for digit arguments.
                 '("1" . meow-digit-argument)
                 '("2" . meow-digit-argument)
                 '("3" . meow-digit-argument)
                 '("4" . meow-digit-argument)
                 '("5" . meow-digit-argument)
                 '("6" . meow-digit-argument)
                 '("7" . meow-digit-argument)
                 '("8" . meow-digit-argument)
                 '("9" . meow-digit-argument)
                 '("0" . meow-digit-argument)
                 '("/" . meow-keypad-describe-key)
                 '("?" . meow-cheatsheet))
                (meow-normal-define-key
                 '("0" . meow-expand-0)
                 '("9" . meow-expand-9)
                 '("8" . meow-expand-8)
                 '("7" . meow-expand-7)
                 '("6" . meow-expand-6)
                 '("5" . meow-expand-5)
                 '("4" . meow-expand-4)
                 '("3" . meow-expand-3)
                 '("2" . meow-expand-2)
                 '("1" . meow-expand-1)
                 '("-" . negative-argument)
                 '(";" . meow-reverse)
                 '("," . meow-inner-of-thing)
                 '("." . meow-bounds-of-thing)
                 '("[" . meow-beginning-of-thing)
                 '("]" . meow-end-of-thing)
                 '("a" . meow-append)
                 '("A" . meow-open-below)
                 '("b" . meow-back-word)
                 '("B" . meow-back-symbol)
                 '("c" . meow-change)
                 '("d" . meow-delete)
                 '("D" . meow-backward-delete)
                 '("e" . meow-next-word)
                 '("E" . meow-next-symbol)
                 '("f" . meow-find)
                 '("g" . meow-cancel-selection)
                 '("G" . meow-grab)
                 '("h" . meow-left)
                 '("H" . meow-left-expand)
                 '("i" . meow-insert)
                 '("I" . meow-open-above)
                 '("j" . meow-next)
                 '("J" . meow-next-expand)
                 '("k" . meow-prev)
                 '("K" . meow-prev-expand)
                 '("l" . meow-right)
                 '("L" . meow-right-expand)
                 '("m" . meow-join)
                 '("n" . meow-search)
                 '("o" . meow-block)
                 '("O" . meow-to-block)
                 '("p" . meow-yank)
                 '("q" . meow-quit)
                 '("Q" . meow-goto-line)
                 '("r" . meow-replace)
                 '("R" . meow-swap-grab)
                 '("s" . meow-kill)
                 '("t" . meow-till)
                 '("u" . meow-undo)
                 '("U" . meow-undo-in-selection)
                 '("v" . meow-visit)
                 '("w" . meow-mark-word)
                 '("W" . meow-mark-symbol)
                 '("x" . meow-line)
                 '("X" . meow-goto-line)
                 '("y" . meow-save)
                 '("Y" . meow-sync-grab)
                 '("z" . meow-pop-selection)
                 '("'" . repeat)
                 '("<escape>" . ignore)))
        (meow-setup)
        (meow-global-mode 1))
```

**Comment by eshrh** [comment]
Sorry, i can't reproduce this on emacs 29. Do you still have this issue in [emacs -Q](https://github.com/meow-edit/meow/blob/master/GET_STARTED.org#try-without-installation)?

**Comment by artistu** [comment]
No, I don't, it works correctly when I only load **meow**. Hrm, is strange, the problem seems to be in:
```
(defun meow-line (n &optional expand)
...
  (let* ((orig (mark))
         (n (if (meow--direction-backward-p)
...
```
According to the documentation **(mark)**, in Transient Mark Mode, signals an error if the mark is not active, which seems to be what is happening to me. If the argument FORCE is non-nil (aka **(mark t)**) it disregards weather the mark is active or not and returns an integer or nil. So, in my case, by changing **(mark)** to **(mark t)** in **meow-line** makes the command behave as expected. 

So in summary, my problem seems to be that something is making NORMAL state to be in Transient Mark Mode, not sure what or how to debug it. I might redefine **meow-line** in my config for the time being.
